### DISCUSSION TITLE
Increasing the size of user_attribute and group_attribute

### DISCUSSION THREAD

**Discussion by khaldounaissaoui2** [keycloak/keycloak#8593] [Archive]
Hello,

There is a limitation for the size of the column value in the two tables  user_attribute and group_attribute.
wich is 255 character.

Would it better to    change the column type to something like :
     **# ALTER TABLE user_attribute ALTER COLUMN value TYPE TEXT;
    # ALTER TABLE group_attribute ALTER COLUMN value TYPE TEXT;**

The usage of the attributes is very powerfull to customize permissions  of a user.

**Comment by hmlnarik** [comment]
Not in the current store, please see https://issues.redhat.com/browse/KEYCLOAK-9515 for the arguments.

**Comment by iudex-** [comment]
I know some time passed, but your post gave me a pointer to fix this im my keycloak instance :slightly_smiling_face:
Thanks!

The command was almost fine, just `SET DATA TYPE` instead of `TYPE` (at least in case the build-in h2 database is being used; [docs](https://www.h2database.com/html/commands.html#alter_table_alter_column))

More details [here](https://github.com/keycloak/keycloak/issues/9758#issuecomment-1092613610)
